node/events_test.ts

```diff
@@ -0,0 +1,1138 @@
+import { assert } from "../testing/asserts.ts";
+import { EventEmitter, getEventListeners, on, once } from "./events.ts";
+import { assertCallbackErrorUncaught } from "./_utils.ts";
+import type { EventListenerT } from "./events.ts";
+import { expectType } from "https://raw.githubusercontent.com/TypeStrong/ts-expect/736658bd9f1c23ad9e2676c27e9bdc9297309ae9/src/index.ts";
+import type { TypeEqual } from "https://raw.githubusercontent.com/TypeStrong/ts-expect/736658bd9f1c23ad9e2676c27e9bdc9297309ae9/src/index.ts";
```

Comment:
We prefer to avoid external dependencies in std. Can you copy and paste these utils to somewhere in std (inlining them in this file also works) as testing-only utilities?